### ISSUE TITLE
Start fails when using ember-truth-helpers 4.x

### ISSUE THREAD

**Issue by sebastianhelbig** [cibernox/ember-power-select#1600]
- ember-power-select: 7.1.2
- ember-auto-import: 2.6.3
- ember-truth-helpers: 4.0.3

```
ember-power-select needs to depend on ember-auto-import in order to use ember-truth-helpers

ERROR Summary:
  - broccoliBuilderErrorStack: [undefined]
  - code: [undefined]
  - codeFrame: [undefined]
  - errorMessage: ember-power-select needs to depend on ember-auto-import in order to use ember-truth-helpers
  - errorType: [undefined]
  - location:
    - column: [undefined]
    - file: [undefined]
    - line: [undefined]
  - message: ember-power-select needs to depend on ember-auto-import in order to use ember-truth-helpers
  - name: Error
  - nodeAnnotation: [undefined]
  - nodeName: [undefined]
  - originalErrorMessage: [undefined]
  - stack: Error: ember-power-select needs to depend on ember-auto-import in order to use ember-truth-helpers
    at Class.registerV2Addon (…/node_modules/.pnpm/@embroider+addon-shim@1.8.6/node_modules/@embroider/addon-shim/src/index.js:122:27)
    at Class.included (…/node_modules/.pnpm/@embroider+addon-shim@1.8.6/node_modules/@embroider/addon-shim/src/index.js:42:18)
    at Class.superWrapper [as included] (…/node_modules/.pnpm/core-object@3.1.5/node_modules/core-object/lib/assign-properties.js:34:20)
    at …/node_modules/.pnpm/ember-cli@5.3.0/node_modules/ember-cli/lib/models/addon.js:475:26
    at Array.reduce (<anonymous>)
    at Class.eachAddonInvoke (…/node_modules/.pnpm/ember-cli@5.3.0/node_modules/ember-cli/lib/models/addon.js:472:24)
    at Class.included (…/node_modules/.pnpm/ember-cli@5.3.0/node_modules/ember-cli/lib/models/addon.js:735:10)
    at Class.superWrapper [as included] (…/node_modules/.pnpm/core-object@3.1.5/node_modules/core-object/lib/assign-properties.js:34:20)
    at Class.included (…/node_modules/.pnpm/ember-power-select@7.1.2_@babel+core@7.23.0_ember-source@5.3.0_webpack@5.88.2/node_modules/ember-power-select/index.js:24:26)
    at Class.superWrapper [as included] (…/node_modules/.pnpm/core-object@3.1.5/node_modules/core-object/lib/assign-properties.js:34:20)
```

**Comment by mkszepp** [comment]
@sebastianhelbig we have already prepared everything for `ember-truth-helpers` v4 to solve this issue (https://github.com/cibernox/ember-power-select/pull/1593).

@cibernox can you make a release for `ember-power-select`?

The latest released version of `ember-power-select` (7.1.2) has dependency `ember-truth-helpers` v3. The next release allows v3 & v4.

**Comment by kiwi-josh** [comment]
Bump on this please 🙏

**Comment by mkszepp** [comment]
@sebastianhelbig @kiwi-josh released in v7.2.0
